From 4627418505bd87b7d8a60f017c9779f0e2f2a928 Mon Sep 17 00:00:00 2001
From: Davide Cavalca <davide@cavalca.name>
Date: Thu, 4 Jul 2024 07:30:57 -0700
Subject: [PATCH] hwdb: add more AV controllers

(cherry picked from commit c1eb4cdd3b933afef2fb9f6f14ba0f9383e08afe)

Related: RHEL-5950
---
 hwdb.d/70-av-production.hwdb | 36 ++++++++++++++++++++++++++++++++++++
 1 file changed, 36 insertions(+)

diff --git a/hwdb.d/70-av-production.hwdb b/hwdb.d/70-av-production.hwdb
index a701d1c1ea..91c757bcb1 100644
--- a/hwdb.d/70-av-production.hwdb
+++ b/hwdb.d/70-av-production.hwdb
@@ -18,6 +18,21 @@
 usb:v2982p1967*
  ID_AV_PRODUCTION_CONTROLLER=1
 
+################
+# Contour
+################
+# Shuttle Pro
+usb:v0B33p0011*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
+# Shuttle Pro V2
+usb:v0B33p0030*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
+# ShuttleXpress
+usb:v0B33p0020*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
 ################
 # Eks
 ################
@@ -83,6 +98,13 @@ usb:v06F8pB100*
 usb:v06F8pB107*
  ID_AV_PRODUCTION_CONTROLLER=1
 
+####################
+# ICOM
+####################
+# RC-28 Remote Encoder
+usb:v0C26p001E*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
 #####################
 # Native Instruments
 #####################
@@ -161,3 +183,17 @@ usb:v17CCp1130*
 # CDJ 2000 NXS 2
 usb:v2B73p0005*
  ID_AV_PRODUCTION_CONTROLLER=1
+
+####################
+# SunSDR
+####################
+# E-Coder 2 Controller
+usb:v1FC9p0003*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
+####################
+# Xencelabs
+####################
+# Quick Keys
+usb:v28BDp5202*
+ ID_AV_PRODUCTION_CONTROLLER=1
